web/src/components/hooks/useDownloadValues.js

```diff
@@ -40,9 +43,15 @@ const useDownloadValues = ({
     try {
       setIsDownloading(true);
       setError(null);
-      const { data } = await _getValues({
+      const { data, error: _error } = await _getValues({
```

Comment:
I wasn't sure how to download a file with `react-query` so I didn't update this hook yet. Will keep looking for examples and come back and refactor this hook.